### PR TITLE
Organization Changes

### PR DESCRIPTION
**Pull request by rdranathunga**
### Related Issues
- Issue `#1` or (None)

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- Related PR `#1` or (None)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by yasinmiran** [comment]
cc:// @brionmario @thivi @DimalChandrasiri @NipuniBhagya @thanujalk @jeradrutnam

File: apps/console/src/features/core/components/organisation-advanced-search.tsx [hunk 1]
```diff
@@ -0,0 +1,802 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable sort-keys */
```
Comment (thivi) [inline apps/console/src/features/core/components/organisation-advanced-search.tsx:2]: Shall we remove these comments?

File: apps/console/src/features/core/components/organisation-advanced-search.tsx [hunk 2]
```diff
@@ -0,0 +1,802 @@
...
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline apps/console/src/features/core/components/organisation-advanced-search.tsx:4]: The year should be 2021

File: apps/console/src/features/core/components/pagination.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
+/* eslint-disable max-len */
```
Comment (thivi) [inline apps/console/src/features/core/components/pagination.tsx:1]: Shall we remove this line and add a license header?

File: apps/console/src/features/data/country.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable object-curly-spacing */
```
Comment (thivi) [inline apps/console/src/features/data/country.js:1]: Shall we remove these comments disabling eslint? Check the other places as well.

**Review by thivi — CHANGES_REQUESTED** [review]

**Comment by brionmario** [comment]
Closing since there is an ongoing effort on some improvements on top of this PR

**Pull request closed without merging**